frontend/taipy-gui/src/components/Taipy/Field.tsx

```diff
@@ -75,6 +77,19 @@ const Field = (props: TaipyFieldProps) => {
                 <span className={className} id={id} style={style}>
                     {value}
                 </span>
+            ) : mode == 'latex' ? (
+                <Suspense fallback={<div>Loading LaTex...</div>}>
+                    <MathJaxContext config={{
```

Comment:
I think the config object should be defined as a const outside of the component

Reply:
Sure I'll go ahead and move it outside of the compoenent